src/handlers/flow/FlowStreamHandler.h

```diff
@@ -79,13 +79,17 @@ class FlowMetricsBucket final : public visor::AbstractMetricsBucket
         TopN<std::string> topDstIP;
         TopN<uint16_t> topSrcPort;
         TopN<uint16_t> topDstPort;
+        TopN<std::string> topSrcIPandPort;
+        TopN<std::string> topDstIPandPort;
         TopN<uint32_t> topInIfIndex;
         TopN<uint32_t> topOutIfIndex;
         topns(std::string metric)
             : topSrcIP("flow", "ip", {"top_src_ips_" + metric}, "Top source IP addresses by " + metric)
             , topDstIP("flow", "ip", {"top_dst_ips_" + metric}, "Top destination IP addresses by " + metric)
             , topSrcPort("flow", "port", {"top_src_ports_" + metric}, "Top source ports by " + metric)
             , topDstPort("flow", "port", {"top_dst_ports_" + metric}, "Top destination ports by " + metric)
+            , topSrcIPandPort("flow", "ip_port", {"top_src_ips_and_port_" + metric}, "Top source IP addresses and port by " + metric)
```

Comment:
@weyrick  @rboucher-ns1  do you agree with this tag name "ip_port" instead of only "ip"